### PR TITLE
Update to balena lint

### PR DESCRIPTION
**Pull request by Page-**
Depends on https://github.com/balena-io/ci-images/pull/51

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -88,12 +89,11 @@
     "jsdom": "^11.6.2",
     "lint-staged": "^4.1.1",
     "match-media-mock": "^0.1.1",
-    "prettier": "^1.15.3",
+    "prettier": "^2.0.5",
```
Comment (sradevski) [inline package.json:92]: Doesn't balena/lint include prettier as well?

Comment (Page-) [reply package.json:92]: It does which is why I updated this to match, the problem is this repo uses eslint for js files with quite a specialized config, so whilst I could switch to balena-lint for the js I don't know enough about the special stuff this repo uses to judge if it's safe and so we need to keep prettier to do `prettier && eslint`

Comment (sradevski) [reply package.json:92]: @Page- we should define react-specific linting rules in @balena/lint and remove everything custom here. I have an issue in the dashboard for it already, but haven't found the time to do it, so for now let's just merge this as it is.

**Comment by Page-** [comment]
@balena-ci retest

**Comment by Page-** [comment]
@balena-ci retest

**Comment by Page-** [comment]
@balena-ci retest

**Comment by Page-** [comment]
@balena-ci retest

**Comment by Page-** [comment]
@balena-ci retest

**Comment by Page-** [comment]
@balena-ci retest

**Comment by Page-** [comment]
@balena-ci retest

**Comment by Page-** [comment]
@balena-ci retest

**Comment by Page-** [comment]
@balena-ci retest

**Comment by Page-** [comment]
@balena-ci retest

**Comment by Page-** [comment]
@balena-ci retest

**Pull request merged**